physics/GFS_SCNV_generic.F90

```diff
@@ -209,6 +210,15 @@ subroutine GFS_SCNV_generic_post_run (im, levs, nn, lssav, ldiag3d, qdiag3d, &
         endif
       endif
 
+      if (rrfs_smoke) then
+        do k=1,levs
+          do i=1,im
+            tem  = (gq0_water_vapor(i,k)-save_qv(i,k)) * frain
```

Comment:
Most of the configurations fail to build because gq0_water_vapor and save_qv are not defined.